packages/client/src/index.ts

```diff
@@ -9,3 +9,5 @@ export {
 } from './client4';
 
 export {TelemetryHandler} from './telemetry';
+
+export {default as WebSocketClient, WebSocketMessage} from './websocket';
```

Comment:
Two default exports in one module?

Reply:
This file doesn't have a default export itself. It's taking the default exports of the other files and giving them concrete names. It's the same as
```ts
import WebSocketClient, {WebSocketMessage} from './websocket';
export {WebSocketClient, WebSocketMessage};
```

It does definitely look a bit weird though. I forget why those clients are default exports instead of named ones to begin with.

Reply:
Got it. I read it backwards for some reason. It's `default as WebSocketClient` not `WebSocketClient as default` 🤦